src/main/java/bhs/devilbotz/Constants.java

```diff
@@ -24,4 +24,12 @@ public final class Constants {
   public static class OperatorConstants {
     public static final int DRIVER_CONTROLLER_PORT = 0;
   }
+
+  public static final class DriveConstants {
+    public static final int kMotorPortLeftMaster = 3;
```

Comment:
change this to CAN ID, also try to avoid the Hungarian notation (the k in the variable)
eg (leftMasterCANID)

Reply:
same with the other 3

Reply:
For context, there are no motor ports, it's all handled through assigned CAN IDs

Reply:
Interesting that most of the "examples" use Hungarian notation. I prefer camel case, but I'll change it to follow the google style conventions.

I'll switch to "MOTOR_LEFT_MASTER_CAN_ID"